xml/mysql_banners.xml

```diff
@@ -0,0 +1,812 @@
+<?xml version="1.0"?>
+<fingerprints>
```

Comment:
Thanks, I'll address both as well as all of the outstanding issues. I hope to have the final PR for review by the end of this coming weekend.